chassis.yaml

```diff
@@ -0,0 +1,10 @@
+# Extension version.
+#
+# All modern extensions should use 2 here. Version 1 uses a chassis.pp
+# instead, which requires a deprecated feature of Puppet.
+#
+# Values: 2
+version: 2
+
+dependencies:
+  - query-monitor
```

Comment:
Negative Sir! If I change that we get: 
![query](https://user-images.githubusercontent.com/1377956/28202260-dd9ddb36-68b7-11e7-9bbf-0d67edbfcd74.png)